dash_bio/component_factory/_clustergram.py

```diff
@@ -201,125 +191,13 @@ def Clustergram(
 
 
 class _Clustergram():
-    """A Dash Bio Clustergram class.
```

Comment:
If we're removing this, we should make reference to the docstring for the "wrapper" `Clustergram()` function above. All of the arguments for that function except `generate_curves_dict`, `return_computed_traces`, and `computed_traces` are eventually passed here.

Reply:
Right, I double-checked every "keyword argument;" except for `generate_curves_dict`, `return_computed_traces`, and `computed_traces` indeed, they are parameters of `_Clustergram()`'s **methods**, so they could/should be documented in each corresponding method's docstring. Would you like me to do this in this PR, or in a different PR?

Reply:
Let's do this in a different PR.

Reply:
Ok.

Reply:
addressing in PR #401